src/constants/EventShellTemp.js

```diff
@@ -0,0 +1,9 @@
+export const eventShellTemporary = {
+    name: "fun event",
```

Comment:
I know this eventShell is just a placeholder, but it you might want to update it to: `name: {text: "fun event"};`. Name is not currently rendering.